pkg/scheduler/framework/interface.go

```diff
@@ -160,6 +162,29 @@ func (s *Status) AsError() error {
 	return errors.New(s.Message())
 }
 
+// StatusEqual returns whether two Statuses are equal to each other.
+func StatusEqual(x, y *Status) bool {
+	if x == y {
+		return true
```

Comment:
In most cases, we won't pass in two identical pointers, right? If so, this `if` block can be removed.

Reply:
I agree